settings/settings_manager.py

```
@@ -0,0 +1,33 @@
+import json
+from pathlib import Path
+
+CONFIG_PATH = Path.home() / ".image_downloader_config.json"
+
+def load_settings(self):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)